core/dungeon-masters-guide/items/items-wondrous.xml

```diff
@@ -4085,4 +4085,483 @@
+			</ul>
+			<p class="indent"><i><b>Properties of the Eye.</b></i> Your alignment changes to neutral evil, and you gain the following benefits:</p>
+			<ul>
+				<li>You have truesight</li>
+				<li>You can use an action to see as if you were wearing a <i>ring of X-ray vision</i>. You can end this effect as a bonus action.</li>
+				<li>The eye has 8 charges. You can use an action and expend 1 or more charges to cast one of the following spells (save DC 18) from it: <em>clairvoyance</em> (2 charges), <em>crown of madness</em> (1 charge), <em>disintegrate</em> (4 charges), <em>dominate monster</em> (5 charges), or <em>eyebite</em> (4 charges). The eye regains 1d4 + 4 expended charges daily at dawn. Each time you cast a spell from the eye, there is a 5 percent chance that Vecna tears your soul from your body, devours it, and then takes control of the body like a puppet. If that happens, you become an NPC under the DM's control.</li>
+			</ul>
+			<p class="indent"><i><b>Properties of the Eye and Hand.</b></i> If you are attuned to both the hand and eye, you gain the following additional benefits:</p>
+			<ul>
+				<li>You are immune to disease and poison.</li>
+				<li>Using the eye's X-ray vision never causes you to suffer exhaustion.</li>
+				<li>You experience premonitions of danger and, unless you are incapacitated, can't be surprised.</li>
+				<li>If you start your turn with at least 1 hit point, you regain 1d10 hit points.</li>
+				<li>If a creature has a skeleton, you can attempt to turn its bones to jelly with a touch of the <em>Hand of Vecna</em>. You can do so by using an action to make a melee attack against a creature you can reach, using your choice of your melee attack bonus for weapons or spells. On a hit, the target must succeed on a DC 18 Constitution saving throw or drop to 0 hit points.</li>
+				<li>You can use an action to cast <em>wish</em>. This property can't be used again until 30 days have passed.</li>
+			</ul>
+			<p class="indent"><i><b>Destroying the Eye and Hand.</b></i> If the <em>Eye of Vecna</em> and the <em>Hand of Vecna</em> are both attached to the same creature, and that creature is slain by the <em>Sword of Kas</em>, both the eye and the hand burst into flame, turn to ash, and are destroyed forever. Any other attempt to destroy the eye or hand seems to work, but the artifact reappears in one of Vecna's many hidden vaults, where it waits to be rediscovered.</p>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_ARTIFACT_TABLE" />
+			</div>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_MAGIC_ITEM_RING_OF_XRAY_VISION" />
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Items</set>
+			<set name="rarity">Artifact</set>
+			<set name="attunement">true</set>
+		</setters>
+		<rules>
+			<grant type="Vision" id="ID_VISION_TRUESIGHT" />
```

Comment:
Due to vision rework we need to assign a range to this or the sheet will just say 0 ft.  Since the item references Ring of Xray Vision then its range of 30ft is probably correct

Reply:
So ring of X-ray vision isn't actually truesight, so instead i feel like just leaving it as a referenced element instead of granting truesight is the right way to do it